src/ostree/ot-admin-builtin-status.c

```diff
@@ -204,6 +207,11 @@ ot_admin_builtin_status (int argc, char **argv, OstreeCommandInvocation *invocat
     {
       g_print ("No deployments.\n");
     }
+  else if (opt_query_booted)
+    {
+      const gboolean is_default_booted = deployments->pdata[0] == booted_deployment;
+      g_print ("%s\n", is_default_booted ? "default" : "not-default");
```

Comment:
I had a:

```
      if (!is_default_booted)
        return FALSE;
```

here also initially, but this causes it to coredump and everything:

```
ostree admin status -Q
not-default
**
OSTree:ERROR:src/ostree/ot-main.c:323:ostree_run: assertion failed: (success || error)
Bail out! OSTree:ERROR:src/ostree/ot-main.c:323:ostree_run: assertion failed: (success || error)
Aborted (core dumped)
```

would have been nice because you could check the exit code of the binary, than a string comparison for "default" in systemd.

Reply:
The classic problem with using exit codes like that then is there's no way to distinguish "failed to compute state because we encountered e.g. an I/O error or the process segfaulted" from "not default".

Reply:
Done